sass/themes/frost/components/_form.scss

```diff
@@ -189,6 +200,35 @@ form {
   background: rgba(255, 255, 255, 0.7);
   width: 100%;
   height: 100%;
+  z-index: 1;
+}
+
+.form__group--age_selection {
+  .age_selection {
+    width: 100%;
+    border-top: 1px solid $colour-border-grey;
+    padding-top: 20px;
+  }
+
+  .btn {
+    width: 45%;
+    @extend .btn--blue-ghost;
+
+    &.active { 
+      background-color: $colour-blue;
+      color: $colour-white;
+      border: 0px;
+      padding: 23px 40px 19px;
+    }
+  }
+
+  .btn + .btn {
```

Comment:
@KoalaMango After you explained it to me, it's not a bad practice at all,
but may be adding a comment above will make it more clear